source/gui/settingsDialogs.py

```diff
@@ -1636,31 +1636,53 @@ def makeSettings(self, settingsSizer):
 		settingsSizer.Add(sizer)
 
 		# Translators: The label for the edit field in symbol pronunciation dialog to change the pronunciation of a symbol.
-		changeSizer = wx.StaticBoxSizer(wx.StaticBox(self, wx.ID_ANY, _("Change symbol")), wx.VERTICAL)
+		changeSizer = wx.StaticBoxSizer(wx.StaticBox(self, wx.ID_ANY, _("Change selected symbol")), wx.VERTICAL)
+
+		# Used to ensure that event handlers call Skip(). In the case of handling EVT_KILL_FOCUS, not calling skip can 
+		# cause focus problems for controls. More generally the advice on the wx documentation is: "In general, it is 
+		# recommended to skip all non-command events to allow the default handling to take place. The command events are,
+		# however, normally not skipped as usually a single command such as a button click or menu item selection must 
+		# only be processed by one handler."
+		def SkipEventAndCall(some_func):

```

Comment:
Interesting. I assume the idea of writing a wrapper for this was to eliminate the possibility of someone forgetting to call evt.Skip?